reorganize-2019-01.md

```diff
@@ -0,0 +1,86 @@
+# Reorganization for rstudio::conf 2019-01
+
+The outline below is for day 1 only.
+
+## 01) Two Ways to Teach
+
+-   Introductions
+-   Comparison with movie/play (add comedy improv)
+-   Tell participants there's lots of research (much of which people don't know/act on)
+-   Schedule (for both days)
+-   Survey distinguishing teacher-centered from learner-centered
+    -   Explain that active learning creates a real-time feedback loop in class
+-   Introduce cognitive architecture diagram (revisit it several times)
+    -   Teaching is "load into short-term memory and keep it there long enough to be transferred"
+-   Bloom's Taxonomy
+    -   Use revised version from early 2000s
+    -   Google isn't enough if you don't know what to search for or how to recognize a useful answer when you find one
+-   Exercise: classify several learning objectives by level
+
+## 02) Novice/Competent Practitioner/Expert
+
+-   Mental models
+    -   Emphasize linkages: we rarely forget completely, but we lose the ability to retrieve
+-   Mental models don't have to be right to be useful (ball-and-spring models in chemistry)
+-   Concept maps as visual representations of mental models
+-   Using concept maps:
+    -   To design lesson
+    -   To convey information to learners
+    -   For insight into their learning
+-   Exercise: draw and compare concept maps for ggplot2
+
+## Coffee
+
+## 03) Teaching as Performance (I)
+
+-   Critique the Nederbragt videos
+    -   https://youtu.be/bXxBeNkKmJE
+    -   https://youtu.be/SkPmwe_WjeY
+-   Exercise: teach in pairs (no recording)
+-   Build rubric for use in afternoon
```

Comment:
Rubric for critiquing teaching?

Reply:
Yup - I've found in the past that if we collate participants' feedback in the first round of live teaching, then have them use that as a rubric in the second, they're more invested.